MuxExoPlayer/src/r2_13/java/com/mux/stats/sdk/muxstats/ima/MuxImaAdsLoader.java

```diff
@@ -155,27 +140,31 @@ public Builder setImaSdkSettings(ImaSdkSettings imaSdkSettings) {
     }
 
     /**
-     * Sets a listener for ad errors that will be passed to {@link
+     * Adds a listener for ad errors that will be passed to {@link
      * com.google.ads.interactivemedia.v3.api.AdsLoader#addAdErrorListener(AdErrorListener)} and
      * {@link AdsManager#addAdErrorListener(AdErrorListener)}.
      *
      * @param adErrorListener The ad error listener.
      * @return This builder, for convenience.
      */
-    public Builder setAdErrorListener(AdErrorListener adErrorListener) {
-      this.adErrorListener = checkNotNull(adErrorListener);
+    public Builder addAdErrorListener(AdErrorListener adErrorListener) {
```

Comment:
Oh and one for `setAdEventListener` also!

Reply:
Do we need those if we introduce a new class, the old clients can still use ImaAds as they did up until now, it will not break the old code, but if they want a multiple event listeners to be registered they can use MuxImaAds classes.

I am saying this does not break the backward compatibility

Reply:
Great point! We wouldn't need `@Deprecated` methods on `MuxImaAdsLaoder` if we still had `ImaAdsLoader`, you are correct